src/components/screens/EventRecord.js

```diff
@@ -44,6 +45,18 @@ export const EventRecord = ({ data }) => {
     title,
     webUrls
   } = data;
+  const link = webUrls && webUrls.length && webUrls[0].url;
+  const rootRouteName = navigation.getParam('rootRouteName', '');
+  // action to open source urls
+  const openWebScreen = (webUrl) =>
```

Comment:
every component has it's special `openWebScreen` method, because there are different params. here for example for `EventRecord` @SoleCincis